src/main/java/com/cascadebot/cascadebot/commands/moderation/BanCommand.java

```diff
@@ -15,20 +15,19 @@
 
     @Override
     public void onCommand(Member sender, CommandContext context) {
-        String banLength;
 
-        if (context.getArgs().length == 0) {
+        if (context.getArg(0) == null) {
```

Comment:
You should still do the length check, the null check is on the `targetMember` variable